src/main/java/org/example/springintro/model/Book.java

```diff
@@ -15,8 +15,8 @@
 @Entity
 @Setter
 @Getter
-@SQLDelete(sql = "UPDATE books SET is_delete = true WHERE id=?")
-@SQLRestriction(value = "is_delete=false")
+@SQLDelete(sql = "UPDATE books SET is_delete = TRUE WHERE id = ?")
+@SQLRestriction(value = "is_delete = FALSE")
```

Comment:
```suggestion
@SQLRestriction(value = "is_deleted = FALSE")
```

rename field to isDeleted